bots/images/scripts/rhel.setup

```diff
@@ -273,8 +273,8 @@ EOF
     fi
 
     useradd -c Builder -G mock builder
-    srpm=$(/var/lib/testvm/make-srpm $TEST_SOURCE)
-    su builder -c "/usr/bin/mock --verbose --installdeps $srpm"
+    su builder -c "/usr/bin/mock --no-bootstrap-chroot --verbose -i $(curl https://raw.githubusercontent.com/cockpit-project/cockpit/master/tools/cockpit.spec |\
+                   sed -n '/^BuildRequires:/ {s/^.*: //; s/>=.*$//; s/%{.*}//; s/\([^ ]*(.*)[^ ]*\)/"\1"/; p}' | tr '\n' ' ')"
```

Comment:
I  actually thought about something like bots/images/scripts/lib/build-deps.sh, which prepares the list of packages, and can then be copied into the VM and re-used between rhel.setup and fedora.setup. It can then also be some more lines, to make it easier to read. But first and foremost, it can easily be tested standalone without having to trigger a package build.